zipkin-storage/cassandra3/src/main/resources/cassandra3-schema.cql

```diff
@@ -1,88 +1,79 @@
-CREATE KEYSPACE IF NOT EXISTS zipkin3 WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '1'};
+CREATE KEYSPACE IF NOT EXISTS zipkin2_cassandra3 WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '1'};
```

Comment:
in this file i still have to add some comments, and i want to duplicate all comments into the schema's table-level ` WITH COMMENT = '…';` so that operators get that information too…

Reply:
done.